src/integrationtests/java/com/aws/greengrass/integrationtests/ipc/AuthorizeClientDeviceActionTest.java

```diff
@@ -146,13 +151,18 @@ void GIVEN_brokerWithUnauthorizedAction_WHEN_AuthorizeClientDeviceAction_THEN_re
                     .withOperation("some-invalid-action")
                     .withResource("some-resource")
                     .withClientDeviceAuthToken("some-token");
-            when(deviceAuthClient.canDevicePerform(any())).thenReturn(false);
+            AuthorizationRequest authzReq = AuthorizationRequest.builder()
```

Comment:
Remove this one though.

Only mock the behavior you want to test. You've already tested that the correct arguments are passed. So now you're just interested in showing that IPC behaves correctly when it receives the various responses. The mock should return `false` no matter what. This is just unnecessarily increasing coupling